.travis.yml

```diff
@@ -77,8 +71,8 @@ script: |
         echo "Skipping documentation upload from source other than Cantera/cantera:master"
       fi
   else
-      python3 `which scons` build -j2 VERBOSE=y python2_package=full python3_package=full python2_cmd=/usr/local/opt/python@2/bin/python2 blas_lapack_libs=lapack,blas optimize=n coverage=y
-      python3 `which scons` test
+      scons build -j2 python_cmd=python3 VERBOSE=y python_package=full blas_lapack_libs=lapack,blas optimize=n coverage=y
```

Comment:
Will this build with Python 2 or Python 3 running SCons? If Python 2 (as I suspect, I think this will pick up the system Python), is that intentional? Should it be intentional?

Reply:
OK, agreed. Just wanted to make sure! 😄